packages/regl-worldview/src/camera/CameraStore.test.js

```diff
@@ -135,6 +138,26 @@ describe("camera store", () => {
         store = new CameraStore(undefined, cameraState);
         expect(store.state).toEqual(cameraState);
       });
+
+      it("has missing properties filled in from DEFAULT_CAMERA_STATE", () => {
```

Comment:
#nit tip for writing test descriptions: usually they're most readable and compact when starting with an active verb, e.g. `it("fills in missing properties from DEFAULT_CAMERA_STATE"`.

Reply:
I considered that phrasing, but it didn't make semantic sense to me. Together with the component being described, the test description would be `initial store state fills in missing properties from DEFAULT_CAMERA_STATE`. But the initial store state doesn't fill in missing properties—the camera store does that.